cypress/support/commands.js

```diff
@@ -23,3 +24,31 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', () => {
+  cy.visit('/');
+  cy.get('.navbar-nav a:contains(Login)').click();
+  cy.get('#email').type(Cypress.env('user'));
+  cy.get('#password').type(Cypress.env('password'));
+  cy.get('button[type="submit"]').click();
+})
+
+Cypress.Commands.add('createsANote', data => {
+  cy.contains('Create a new note', { timeout: 30000 }).should('be.visible').click();
+  cy.get('#content').type(data.content);
+  cy.contains('Create').click();
+  cy.get('.list-group').should('contain', data.content);
+})
+
+Cypress.Commands.add('editsANote', data => {
+  cy.get('.list-group').contains(data.content).click();
+  cy.get('#content').clear().type(data.newContent);
+  cy.contains('Save').click();
```

Comment:
For this project, I was not using semi-colons at the end. I noticed that you're using it in some places and not in others.

Let's have a standard (I suggest without it) and use it everywhere.

Reply:
Done!

I removed all of them.